AmazonChimeSDKDemo/AmazonChimeSDKDemo.xcodeproj/project.pbxproj

```diff
@@ -805,7 +805,6 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0.16.6;
 				DEBUG_INFORMATION_FORMAT = dwarf;
-				DEVELOPMENT_TEAM = "";
```

Comment:
looks like this cameback too

Reply:
Fixed. Somehow rebase overwrote it without raising a conflict